uiowa/uiowa
# AoS Follow-up

- changes combined filter parameter from `combine` to `title`. this provides consistency with the search block help text and the view filter label.
- remove area_of_study_search custom block type from feature split (not used and being listed in lb_direct_add)

## To Test

- using the search block and a placed areas of study view, set the parameter to `title` and try a search on the view. the search value should be populated in the view filters.
- you won't see Areas of Study Search as a lb_direct_add option above Articles.
